tests/MimeContentTypeTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+namespace Mmoreram\React\Tests;
+
+use PHPUnit\Framework\TestCase;
+use React\EventLoop\Factory;
+use Mmoreram\React;
+use Clue\React\Block;
+
+/**
+ * Class MimeContentTypeTest
+ */
+class MimeContentTypeTest extends TestCase
+{
+    function testIt()
+    {
+        $loop = Factory::create();
+        $promise1 = React\mime_content_type(__DIR__ . '/static/file1.txt', $loop);
+        $promise2 = React\mime_content_type(__DIR__ . '/static/pixel.png', $loop);
+
+        $results = Block\awaitAll([$promise1, $promise2], $loop);
+        $this->assertEquals('text/plain', $results[0]);
```

Comment:
Actually here you don't test that the promise resolves or rejects. You only validate the returned values. I can suggest using [my library for testing promises](https://github.com/seregazhuk/php-react-promise-testing) where you can write assertions like this:

```php
$this->assertPromiseFulfillsWith($promise1, 'text/plain');
```

The test becomes more readable. And we can assert that the promise has rejected or failed.

Reply:
Oh, sorry. Your code **tests** that the promise resolves. Otherwise `awaitAll()` will throw an exception 🙈